crates/driver/src/infra/cli.rs

```diff
@@ -21,6 +21,10 @@ pub struct Args {
     #[clap(long, env)]
     pub ethrpc: Url,
 
+    /// The base URL to connect to subgraph clients.
+    #[clap(long, env, default_value = "https://api.thegraph.com/subgraphs/name/")]
+    pub graph_api_base_url: Url,
```

Comment:
We should probably not continue requiring the same base url for every use case of the graph and instead pass around the complete url in the univ3 and balancer v2 liquidity section of the config.
Otherwise we are forced to completely move towards self hosting the subgraph on the same base url.